resources/js/components/showhistory.js

```diff
@@ -1,19 +1,24 @@
 import {Link} from 'react-router-dom';
 import { useState, useEffect } from 'react';
 import AppContainer from './AppContainer';
+import Pagination from 'react-js-pagination';
+
 
 const ShowHistory = () =>{
 
     const [results, setResults] = useState([]);
+    const [paginations, setPagination] = useState();
     const [spinner, setSpinner] = useState(false);
 
-    const fetchResult = () => {
+    const fetchResult = async (pageNumber = 1) => {
         setSpinner(true);
 
-        axios.get('http://localhost/api/results')
+        await axios.get('http://localhost/api/results?page='+pageNumber)
         .then(res => {
             setSpinner(false);
-            setResults(res.data);
+            setPagination(res.data);
+            console.log(res);
```

Comment:
You can put message first to make it more meaningful.

console.log(“api results”, res);